cmd/security-secretstore-setup/entrypoint.sh

```diff
@@ -44,6 +44,9 @@ fi
 echo "$(date) Changing ownership of ${EDGEX_SECRETS_ROOT} to ${EDGEX_USER}:${EDGEX_GROUP}"
 chown -Rh ${EDGEX_USER}:${EDGEX_GROUP} "${EDGEX_SECRETS_ROOT}"
 
+echo "$(date) Changing ownership of ${EDGEX_SECRETS_ROOT}/rules-engine to 1001:1001 to accommodate ekuiper"
+chown -Rh 1001:1001 "${EDGEX_SECRETS_ROOT}/rules-engine"
```

Comment:
why shoud we provide 1001 user permission to it?

Reply:
The ekuiper docker container ships with a `kuiper` user assigned to id 1001. Before, kuiper was not secured using the JWT mechanism from vault, but now it needs access to a token in order to access the openziti overlay. 

#### example
```
$ docker run --rm lfedge/ekuiper:1.14 grep kuiper /etc/passwd
kuiper:x:1001:1001:Linux User,,,:/home/kuiper:/sbin/nologin
```

Reply:
Could you claim the ekuiper users with variables as we did for the EdgeX users?  It can also provide the opportunity for users to overwrite via the environment variable.

```
# default User and Group in case never set
if [ -z "${EDGEX_USER}" ]; then
  EDGEX_USER="2002"
  EDGEX_GROUP="2001"
fi
```